src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+@import "../utils/_variables";
+
+@import "../utils/mixins";
+
+@import "../utils/normalize";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-box-size;
+  gap: $box-gap;
+  margin: 100px;
```

Comment:
I don't exactly understand why you use margin of 100px. For me, in this case margin 0 auto will be more suitable